CNAME

```diff
@@ -0,0 +1 @@
+caprivacy.me
```

Comment:
why is this necessary to add a CNAME to my own root this way? Google domains which is where this is registered doesn't support it.

Reply:
Doing the CNAME allows it to look cleaner. Yeah your right Google Domains doesn't support CNAME at root level. Im so used to using Cloudflare for everything.  Cloudflare supports it. Looks like twofactorauth.org is using cloudflare as well to take care of this. If you don't want to do cloudflare you could do a redirect from caprivacy.me to www.caprivacy.me which you can set as a CNAME.

Error message from google domains:
```
CNAME records for the root domain are not supported
```

Seems like other people have this issue.
https://serverfault.com/questions/617248/does-google-domains-support-cname-like-functionality-at-the-zone-apex


If you just want to keep it showing up as https://caprivacy.github.io/caprivacy/ let me know and i can update the asset references in the code to match that base.

Reply:
https://github.com/2factorauth/twofactorauth/blob/master/CNAME